tests/sanity/ignore-2.9.txt

```diff
@@ -1,76 +0,0 @@
-plugins/module_utils/turbo/exceptions.py compile-2.6!skip
```

Comment:
We should leave all of this in the file for 2.9, 2.10 and 2.11. The `tests/config.yml` won't work on these versions of ansible.